internal/resource/factory.go

```diff
@@ -39,7 +41,7 @@ func WithConfig(manager Manager, config *spec.Config) Manager {
 }
 
 // getManager returns the resource manager depending on the system configuration.
-func getManager(config *spec.Config) Manager {
+func getManager(mode *string) Manager {
```

Comment:
This should accept a `string` and not a `*string`

Reply:
Fixed